api/graphite.go

```diff
@@ -47,10 +47,15 @@ type Series struct {
 }
 
 func (s *Server) findSeries(orgId int, patterns []string, seenAfter int64) ([]Series, error) {
-	peers := cluster.MembersForQuery()
+	series := make([]Series, 0)
+	peers, err := cluster.MembersForQuery()
+	if err != nil {
+		log.Error(3, "HTTP findSeries unable to get peers, %s", err)
+		return series, err
```

Comment:
couldn't that just `return nil, err`, then you'd only have to `series := make([]Series, 0)` if there's no error.